gas.asciidoc

```diff
@@ -1,15 +1,15 @@
+As computational "work" is required for programs to perform an action or set of actions, there is a cost associated with executing smart contract code and transacting on the Ethereum blockchain. This cost is related to the type and number of computational steps that are being executed. In contrast to Bitcoin transaction fees, which only take into account the size of a transaction (kB), Ethereum transaction fees must account for an arbitrary number of computational steps that can be performed by smart contract code. The higher the number of operations a program performs, the higher the cost to run it to completion. **Gas** is Ethereum's unit for measuring how much computational "work" a program takes to perform an action or a set of actions. Every operation performed by a transaction or contract costs a certain number of gas.
 
 To each operation, a fixed amount of gas is assigned. Some examples from the Ethereum yellow paper:
 
 * Adding two numbers costs 3 gas
-* Calculating a <<Keccak256>> hash costs 30 gas + 6 more gas for every 256 bits of data being hashed
+* Calculating a Keccak256 hash costs 30 gas + 6 more gas for every 256 bits of data being hashed
 * Sending a transaction costs 21000 gas
 
-Gas is a crucial component of Ethereum's defense against denial of service attacks. In order to prevent accidental or malicious infinite loops or other computational wastage in the network, the initiator of each transaction is required to set a limit to the amount they are willing to spend on gas. The gas system therefore disincentivizes attackers from sending spam transactions, as they must pay proportionately for the computational, bandwidth, and storage resources that they consume.
+Gas is a crucial component of Ethereum's defense against denial of service attacks. In order to prevent accidental or malicious infinite loops or other computational wastage in the network, the initiator of each transaction is required to set a limit to the amount they are willing to spend on gas. The gas system therefore disincentives attackers from sending spam transactions, as they must pay proportionately for the computational, bandwidth, and storage resources that they consume.
```

Comment:
According with Google Translator **"disincentivizes"** was the correct translatio to Spanish.